### ISSUE TITLE
TypeError: Cannot read property '1' of null

### ISSUE THREAD

**Issue by arjunkomath** [santinic/how2#5]
Found this issue while selecting an answer from the list of more answers.

```
Press SPACE for more choices, any other key to quit.
TypeError: Cannot read property '1' of null
    at Object.parseStackoverflowQuestionId (/usr/local/lib/node_modules/how2/lib/utils.js:45:26)
    at /usr/local/lib/node_modules/how2/lib/how2.js:20:32
    at ScrollableBox.<anonymous> (/usr/local/lib/node_modules/how2/lib/ui.js:66:9)
    at ScrollableBox.EventEmitter._emit (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/events.js:94:20)
    at ScrollableBox.EventEmitter.emit (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/events.js:117:12)
    at ScrollableBox.List.enterSelected (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/widgets/list.js:586:8)
    at ScrollableBox.<anonymous> (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/widgets/list.js:119:14)
    at ScrollableBox.EventEmitter._emit (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/events.js:94:20)
    at ScrollableBox.EventEmitter.emit (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/events.js:117:12)
    at Program.<anonymous> (/usr/local/lib/node_modules/how2/node_modules/blessed/lib/widgets/screen.js:602:15)
```

**Comment by santinic** [comment]
thanks man, looking into it

**Comment by Carrigan** [comment]
I also got this error, but from a different stack trace. Thought I'd post it here to help with the debugging :smile:

```
TypeError: Cannot read property '1' of null
    at Object.parseStackoverflowQuestionId (/usr/local/lib/node_modules/how2/lib/utils.js:45:26)
    at /usr/local/lib/node_modules/how2/lib/how2.js:20:32
    at /usr/local/lib/node_modules/how2/lib/how2.js:194:9
    at /usr/local/lib/node_modules/how2/lib/how2.js:108:9
    at Request._callback (/usr/local/lib/node_modules/how2/node_modules/google/lib/google.js:85:7)
    at Request.self.callback (/usr/local/lib/node_modules/how2/node_modules/request/request.js:199:22)
    at emitTwo (events.js:100:13)
    at Request.emit (events.js:185:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/how2/node_modules/request/request.js:1036:10)
    at emitOne (events.js:95:20)
```

**Comment by santinic** [comment]
Thanks guys. Fixed in 1.0.5, update with

npm update -g how2
